internal/k8s/watch.go

```diff
@@ -20,6 +20,42 @@ import (
 	"github.com/windmilleng/tilt/pkg/logger"
 )
 
+var PodGVR = v1.SchemeGroupVersion.WithResource("pods")
+var ServiceGVR = v1.SchemeGroupVersion.WithResource("services")
+var EventGVR = v1.SchemeGroupVersion.WithResource("events")
+
+// A wrapper object around SharedInformer objects, to make them
+// a bit easier to use correctly.
+type ObjectUpdate struct {
+	obj      interface{}
+	isDelete bool
+}
+
+func (r ObjectUpdate) AsPod() (*v1.Pod, bool) {
+	if r.isDelete {
```

Comment:
I want a comment (here? on `AsDeletedKey`? At the top of this file?) that deletion events are processed agnostic to the type of resource they're deleting.

Reply:
added some comments on AsDeletedKey. Basically, the poor informer is trying to do the best it can with limited information